src/OrbitGl/App.cpp

```diff
@@ -1318,7 +1318,7 @@ void OrbitApp::StartCapture() {
   bool enable_introspection = IsDevMode() && data_manager_->get_enable_introspection();
   bool enable_user_space_instrumentation =
       IsDevMode() && data_manager_->enable_user_space_instrumentation();
-  double samples_per_second = data_manager_->samples_per_second();
+  double samples_per_second = IsDevMode() ? data_manager_->samples_per_second() : 1000.0;
```

Comment:
Nit: Maybe move the `1000.0` into a constant on top of the file, makes it easier to spot?

Reply:
I created a local constant as this value is only used here in this file.